internal/ingress/controller/kong.go

```diff
@@ -415,6 +412,12 @@ func (n *KongController) toDeckContent(
 
 	return &content
 }
+
+func sortByString(slice interface{}, fieldFn func(i int) string) {
```

Comment:
on the off chance we put this here and then forget about it, but then someone else wants to implement the same logic somewhere else later would it be worthwhile to move this into `internal/*/utils` or something like that do you think?

Reply:
1. Historically, exporting util functions followed direct intent to reuse, not the other way around. Not saying that it's the only right way - just we've happened to work like that so far. This may be my own bias, but a function being exported or not doesn't affect my ability to remember its existence when I need a similar one elsewhere.
2. This function is a 2-liner. Exporting and importing proactively feels like an overkill to me for this degree of complexity.
3. An exported function would need individual tests. Here, this helper function is tested as a part of a bigger thing.